examples/server-side-rendering/src/app/app.component.ts

```diff
@@ -2,10 +2,29 @@ import { Component, Injector, Inject, PLATFORM_ID } from '@angular/core';
 import { isPlatformServer } from '@angular/common';
 import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { TransferState, makeStateKey } from '@angular/platform-browser';
-import {
-  createSSRSearchClient,
-  parseServerRequest,
-} from 'angular-instantsearch';
+import { history } from 'instantsearch.js/es/lib/routers';
+import { simple } from 'instantsearch.js/es/lib/stateMappings';
+import { StateMapping } from 'instantsearch.js/es/types';
+import { createSSRSearchClient } from 'angular-instantsearch';
+import * as qs from 'qs';
+
+function parseServerRequest(
```

Comment:
We can inline this function in the component. It makes it clearer what `stateMapping` is IMO.